deel-2-flexbox-layout/opdracht-2/styles/styles.css

```diff
@@ -1 +1,40 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+body{
+    display: flex;
+    flex-direction: row;
+    min-height: 100vh;
+}
+
+div{
+    display: flex;
+    flex-direction: column;
+}
+
+main{
+    display: flex;
+    flex-direction: column;
+    flex-basis: auto;
+    min-width: 80vw;
```

Comment:
De min-width is hier overbodig. Als je de flex basis een viewport width had gegeven en de div ook dan had je hetzelfde resultaat behaald.